content/guides/references/configuration.md

```diff
@@ -2,18 +2,24 @@
 title: Configuration
 ---
 
-## Configuration file
+## Configuration File
 
-The first time you open the Cypress App, it creates the `cypress.config.js`
-configuration file. This Javascript file is used to store any configuration
-values you supply. If you
-[configure your tests to record](/guides/dashboard/projects#Setup) the results
-to the [Cypress Dashboard](https://on.cypress.io/dashboard-introduction) the
-`projectId` will be written in this file too.
+Launching the Cypress App for the first time, you will be guided through a
+wizard that will create a Cypress configuration file for you. This file will be
+`cypress.config.js` for JavaScript apps or `cypress.config.ts` for
+[TypeScript](/guides/tooling/typescript-support) apps. This file is used to
+store any configuration specific to Cypress.
 
-Alternately, Cypress supports a [TypeScript](/guides/tooling/typescript-support)
-`cypress.config.ts` configuration file, and a (deprecated) `cypress.json`
-configuration file.
+Cypress additionaly supports config files with `.mjs` or `.cjs` extenstions as
+well.
+
+Using a `.mjs` file will allow you to use
+[ESM Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
+syntax in your config without the need of a tranpiler step.
+
+A '.cjs' file uses the [CommonJS](https://nodejs.org/api/modules.html) module
+sytntax, which is the default for JavaScript files. All JavaScript config
+examples in our docs use the CommonJS format.
```

Comment:
```suggestion
Using a `.mjs` file will allow you to use
[ESM Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
syntax in your config without the need of a transpiler step.

A '.cjs' file uses the [CommonJS](https://nodejs.org/api/modules.html) module
syntax, which is the default for JavaScript files. All JavaScript config
examples in our docs use the CommonJS format.
```

@elylucas I noticed a few typos -- it might be helpful to install a spell checker to your code editor. I use [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for VS Code. It helps a ton!